pallets/msa/src/lib.rs

```diff
@@ -378,15 +401,16 @@ impl<T: Config> Pallet<T> {
 	}
 
 	pub fn revoke_delegate(
-		delegate_msa_id: Delegate,
+		provider_msa_id: Delegate,
```

Comment:
went ahead and started using "provider" here.  I did not want to touch everything else though.